packages/waku/src/router/define-router.ts

```diff
@@ -368,17 +374,41 @@ export function new_defineRouter(fns: {
           has404,
         };
   };
+  const canSkip = async (
+    pathname: string,
+    slotId: SlotId,
+  ): Promise<boolean> => {
+    const pathConfig = await getMyPathConfig();
```

Comment:
Actually, should `canSkip` take the result of `getMyPathConfig()` as a parameter? It seems unnecessary to await it for every pathname checked.

Reply:
it's possible. await just waits one microtask in this case, so I didn't care much.

Reply:
Ok!

Reply:
I don't disagree it's not ideal and there is room for improvement. Let's revisit it later as we will be rewriting this anyway.